### PR TITLE
12.Ansible

### PR DESCRIPTION
**Pull request by Korolev731**

### CODE REVIEW

File: IKorolev/re.md [hunk 1]
```diff
@@ -0,0 +1 @@
+sAF
```
Comment (pluhin) [inline IKorolev/re.md:1]: What is the file?

File: README.md [hunk 2]
```diff
@@ -16,3 +16,4 @@ Main purpose of this repository is collaboration of homework for study groups
 ## CI/CD
 
 In preparing
+# 1
```
Comment (pluhin) [inline README.md:19]: Do not do changes in main readme file

**Review by pluhin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pluhin) [reply README.md:19]: Please remove changes in this readme file

**Review by pluhin — CHANGES_REQUESTED** [review]
Good, approved

File: IKorolev/re.md (hunk 1 above)
Comment (Korolev731) [reply IKorolev/re.md]: fixed

File: README.md (hunk 2 above)
Comment (Korolev731) [reply README.md:19]: fixed

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**